### ISSUE TITLE
launch pluginval, fails two of its own unit tests on startup

### ISSUE THREAD

**Issue by jcooper-korg** [Tracktion/pluginval#55]
pluginval develop branch, commit 149588a.  built with juce 6.1.4, Xcode 13.1. on macOS 11.6 on intel

When starting from Xcode debugger, it stops twice at jassertfalse (but I can continue past them, on to use the program, so it's not a major problem:
```
Starting test: CommandLineTests / Command line random...
!!! Test 1 failed: Expected value: 8334753, Actual value: -8442463
JUCE Assertion failure in juce_UnitTest.cpp:278
!!! Test 2 failed: Expected value: 110279711, Actual value: -158155745
JUCE Assertion failure in juce_UnitTest.cpp:278

FAILED!!  2 tests failed, out of a total of 2
```

**Comment by jcooper-korg** [comment]
Tested ok in JUCE 6.1.0.  bad in 6.1.4

**Comment by drowaudio** [comment]
I think this is fixed in the 1.0.0 release
